samples/Forms/LibVLCSharp.Forms.MediaElement/LibVLCSharp.Forms.Sample.MediaElement.iOS/AppDelegate.cs

```diff
@@ -26,5 +29,10 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
+
+        public override UIInterfaceOrientationMask GetSupportedInterfaceOrientations(UIApplication application, [Transient] UIWindow forWindow)
+        {
+            return OrientationChangeSuscriber.Suscribe(this);
```

Comment:
Not sure about the naming for `OrientationChangeSuscriber` yet. You're missing a `b` though.

`OrientationChangeSubscriber` and `Subscribe` namely.

Are we doing anything with the `this` parameter? Do we really need it?

Reply:
For the naming, OrientationChangeSuscriber listens to the orinetationchange event, that's why I call it so. `b` is missing.
Any proposal for the name ?
`this` is the subscriber that really lock the screen on iOS.